ckanext/canada/tables/contracts.yaml

```diff
@@ -1,31 +1,34 @@
 dataset_type: contracts
 target_dataset: contracts
 
-title: Proactive Disclosure - Contracts over 10K
+title: Proactive Publication - Contracts over 10K
 notes: Access, upload and modify the Contracts over 10K reports for your organization
 
 upload_info:
   en: |
-    The contracts over $10,000 template has recently been updated with new data fields and data validation rules
-    per the Guidelines on the Proactive Disclosure of Contracts. Users should review the data validation rules in the
-    reference tab of the template for each data field under “validation” for more information to ensure that the data can
-    be uploaded to the registry.
+    The contracts over $10,000 template includes data fields that have data
+    validation rules per the Guide on the Proactive Publication of Contracts. Users
+    should review the data validation rules in the reference tab of the template for
+    each data field under “validation” for more information to ensure that the data
+    can be uploaded to the registry. 
   fr: |
-    Le modèle des contrats de plus de 10 000 $ a récemment été mis à jour avec de nouveaux champs de données
-    et des règles de validation des données conformément aux Lignes directrices sur la divulgation proactive des marchés.
-    Les utilisateurs devraient examiner les règles de validation des données dans l’onglet de référence du modèle pour
-    chaque champ de données sous « validation » pour plus d’informations afin de s’assurer que les données peuvent être
-    téléchargées dans le registre.
+    Le modèle pour les contrats de plus de 10 000 $ comprend des champs de
+    données qui sont soumis à des règles de validation des données conformément
+    au Guide de la publication proactive des marchés.
+    Les utilisateurs devraient examiner les règles de validation des données dans
+    l’onglet de référence du modèle pour chaque champ de données sous «
+    validation » pour plus d’informations afin de s’assurer que les données peuvent
+    être téléchargées dans le registre.
 
 template_version: 3
-template_updated: 2023-03-14
+template_updated: 2023-08-14
```

Comment:
August?

Reply:
That is what the client indicated. Copied from the document.